zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Collections.java

```diff
@@ -225,22 +227,22 @@ private void updateDatasetVersionInCollection(Collection collection, String data
         try (InputStream body = request.getInputStream()) {
 
             CollectionDatasetVersion datasetVersion = ContentUtil.deserialise(body, CollectionDatasetVersion.class);
-            datasetService.updateDatasetVersionInCollection(collection, datasetID, edition, version, datasetVersion);
+            datasetService.updateDatasetVersionInCollection(collection, datasetID, edition, version, datasetVersion, user);
 
         } catch (JsonSyntaxException ex) {
             throw new BadRequestException(ex.getMessage());
         }
     }
 
-    private void updateDatasetInCollection(Collection collection, String datasetID, HttpServletRequest request) throws ZebedeeException, IOException, DatasetAPIException {
+    private void updateDatasetInCollection(Collection collection, String datasetID, HttpServletRequest request, String user) throws ZebedeeException, IOException, DatasetAPIException {
```

Comment:
Can you add the user to the logParameters as well please (there is already a `.user(String email)` on the log builder so no need to do `addParameter`.  Can you do this in all places where you have added the user field that is (already logging something).

Reply:
Should be covered in commit https://github.com/ONSdigital/zebedee/pull/138/commits/bd73d27b2039351dfc29e46774658062904d9282